### PR TITLE
Add linkode.el

### PR DESCRIPTION
**Pull request by erickgnavar**
### Brief summary of what the package does
Generate a linkode snippet with region/buffer content

### Direct link to the package repository

https://github.com/erickgnavar/linkode.el

### Your association with the package

Author

### Relevant communications with the upstream package maintainer

None needed

### Checklist

Please confirm with `x`:

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses).
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [ ] I have confirmed some of these without doing them

### CODE REVIEW

**Comment by riscy** [comment]
A very quick first pass --

- linkode.el#L63: No `format` required; messages are already f-strings
- linkode.el#L13: Prefer `https` over `http` if possible ([why?](https://news.ycombinator.com/item?id=22933774))
- Please add license boilerplate or an [SPDX-License-Identifier](https://spdx.org/using-spdx-license-identifier) to linkode.el

**Comment by erickgnavar** [comment]
Hi @riscy, thanks for the suggestions, I applied them to the package.

**Comment by riscy** [comment]
No blockers on this one.

I wonder if your `Commentary` could be improved by adding some of the [usage instructions](https://github.com/erickgnavar/linkode.el) you've got in your README.  I think you could also add a link to the linkode.org site to give viewers of the MELPA index a little more context.

The function `linkode--process-response` has a comment to "fix this" -- I am assuming this is all sorted, but you probably need to update that comment (or reword it to remove the potentially confusing suggestion that something needs to be fixed).

**Comment by erickgnavar** [comment]
Thanks for the feedback, I improved the commentary section and remove the `TODO`
